docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -123,17 +123,17 @@ The default `walkthrough.json` does not define any dependent services:
 ----
 {
   "dependencies": {
-    "repos": [],
+    "/Users/bdooley/Documents/code/user-documentations": [],
     "serviceInstances": []
   }
 }
 ----
 
-=== Committing your {walkthrough-name}s to a remote repository
+=== Committing your {walkthrough-name}s to a remote /Users/bdooley/Documents/code/user-documentationsitory
```

Comment:
This sentence doesn't make sense to me TBH.

Reply:
fixed.